documentation/tutorial.md

```diff
@@ -68,3 +78,660 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploy Charmed MongoDB
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
```

Comment:
I got a bit confused by this, and wasn't sure if I had to go and separately fetch the charm myself because it's linked. It was only after trying running the command that I realised that the command actually does that for me, so I'd reword to something more like the following to make it a bit clearer: